test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go

```diff
@@ -84,7 +84,7 @@ var _ = Describe("[rfe_id:49062][workloadHints] Telco friendly workload specific
 				By("Modifying profile")
 				profile.Spec.WorkloadHints = nil
 
-				if profile.Spec.RealTimeKernel.Enabled == pointer.Bool(false) {
+				if *profile.Spec.RealTimeKernel.Enabled != true {
```

Comment:
So I'm assuming we want to skip the test when 

```
*profile.Spec.RealTimeKernel.Enabled == false 
```

right? Would it be a bit more readable? Similarly below.

Reply:
Right == false would have made more readable. updating the patch